code/modules/client/preference_setup/loadout/loadout_shoes.dm

```diff
@@ -82,4 +82,36 @@
 
 /datum/gear/shoes/tongs/New()
 	..()
-	gear_tweaks = list(gear_tweak_free_color_choice)
\ No newline at end of file
+	gear_tweaks = list(gear_tweak_free_color_choice)
+
+/datum/gear/shoes/hitops/
```

Comment:
Can't you just shove all of those high-top in a single selection, like the flats are handled?